lib/hubspot/config.rb

```diff
@@ -24,12 +24,6 @@ def configure(config)
         @client_secret = config["client_secret"] if config["client_secret"].present?
         @redirect_uri = config["redirect_uri"] if config["redirect_uri"].present?
 
-        unless access_token.present? ^ hapikey.present?
-          Hubspot::ConfigurationError.new("You must provide either an access_token or an hapikey")
```

Comment:
Why was this check removed?

Reply:
because it was not raising anything :S
The it would just enter the if and move along

Reply:
Oh, ha! You're right. That's just a bug with the original implementation though (I wrote it). We just need to put a "fail" in front of it.

Reply:
@adrian-gomez said he tried adding it, but tests broke, and that certain rake tasks don't need the authentication until the time of making the request, and don't set it until then. So just removing.

Reply:
yeah it will still fail when the request its made because of this `Hubspot::Config.ensure! :hapikey` of if the given access_token its invalid